tests/main/classic-confinement-not-supported/task.yaml

```diff
@@ -3,8 +3,7 @@ summary: Ensure that classic confinement works
 environment:
     CLASSIC_SNAP: test-snapd-classic-confinement
 
-# TODO: enable for other systems too?
-systems: [ubuntu-core-1*, fedora-*]
+systems: [ubuntu-core-*, fedora-*, arch-*, centos-*]
```

Comment:
Hm, this fails it seems: `    - google:arch-linux-64:tests/main/classic-confinement-not-supported`